v2/internal/testcommon/resource_groups.go

```diff
@@ -0,0 +1,27 @@
+/*
+Copyright (c) Microsoft Corporation.
+Licensed under the MIT license.
+*/
+
+package testcommon
+
+import (
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	resources "github.com/Azure/azure-service-operator/v2/api/microsoft.resources/v1alpha1api20200601"
+)
+
+func CreateResourceGroup() *resources.ResourceGroup {
```

Comment:
minor: Maybe combine this file, `resources.go` and `scheme.go` together? They seem a little bitsy like this.

Reply:
I couldn't think of a good name for such a file. Suggestions?

Reply:
I'll merge them in a later PR, pending good suggestions on naming.